src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/IssueAttribute.cs

```diff
@@ -22,6 +23,9 @@ namespace Infrastructure.Common
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
     public class IssueAttribute : WcfSkippableAttribute
     {
+        private static bool s_includeAllIssues = DetectIssuesToInclude();
+        private static HashSet<int> s_includeIssuesHash;
```

Comment:
+1. I was reading through this code and it took a while to figure out what the state of s_includeIssuesHash was and how it impacted skipping/inclusion logic